webui/react/src/pages/FlatRuns/FlatRuns.tsx

```diff
@@ -542,22 +517,18 @@ const FlatRuns: React.FC<Props> = ({ projectId, workspaceId, searchId }) => {
     if (isLoadingSettings || Loadable.isNotLoaded(loadableFormset)) return;
     try {
       const filters = JSON.parse(filtersString);
+      const filterFormSet = JSON.parse(filtersString) as FilterFormSetWithoutId;
```

Comment:
this doesn't seem to be sent to the search?

at any rate, given that we do this again for the flatRunActions button, maybe it would make sense to observe `filterFormSetWithoutId` and use a memo to combine it and the search filter and stringify it to pass here and in the actions button?